src/test/java/com/acrolinx/sidebar/pojo/document/CheckContentTest.java

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2018-present Acrolinx GmbH */
+package com.acrolinx.sidebar.pojo.document;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class CheckContentTest
+{
+
```

Comment:
nit-picking: unnecessary empty line, also in a few other classes below

you can use the following regular expression to quickly find them all:

`\{(\s*\n){2,}`